nixos/modules/hardware/cpu/intel-opencl.nix

```diff
@@ -0,0 +1,31 @@
+{
+  config,
+  lib,
+  pkgs,
+  ...
+}:
+{
+  options.hardware.intel.opencl.runtime = lib.mkOption {
+    type = lib.types.nullOr (
+      lib.types.enum [
+        "intel-compute-runtime"
+        "intel-ocl"
+        "beignet"
+      ]
+    );
+    default = null;
+    description = lib.mdDoc ''
```

Comment:
Please remove the occurrence(s) of `lib.mdDoc`

Reply:
I was confused by grepping the old checkout here...